modules/microshift-cni.adoc

```diff
@@ -0,0 +1,271 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-understanding networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-cni_{context}"]
+= About the OVN-Kubernetes CNI network provider
+
+[IMPORTANT]
+The default CNI configuration is intended to match the developer environment described in {product-title} Development Environment on RHEL 8.
+
+//MicroShift Development Environment on RHEL 8 will need a module somewhere to link to, preferably within this assembly; or as an Additional Resource
+
+OVN-Kubernetes is the default networking solution for {product-title} deployments. OVN-Kubernetes is a virtualized network for pods and services that is based on Open Virtual Network (OVN). The OVN-Kubernetes Container Network Interface (CNI) plug-in is the network provider for the cluster. A cluster that uses the OVN-Kubernetes network provider also runs Open vSwitch (OVS) on the node. OVN configures OVS on the node to implement the declared network configuration.
+
+[id="microshift-network-topology_{context}"]
+== Network topology
+OVN-Kubernetes provides an overlay-based networking implementation. This overlay includes an OVS-based implementation of Service/ServiceIP and NetworkPolicy. The overlay network uses the geneve tunnel, so the pod maximum transmission unit (MTU) is set to smaller than that of the physical interface on the host to remove the tunnel header.
```

Comment:
I am not sure if it makes sense to talk about it being an overlay-based networking ... if we don't really use that feature (multi node).

```suggestion
OVN-Kubernetes provides the networking implementation. This includes an OVS-based implementation of Service and NetworkPolicy.
```

? @zshi-redhat wdyt?

Reply:
We have overlay mentioned in one other place (line 177).  If overlay is removed here, it shall also be removed from line 177.

Re the tunnel, it gives a reason for why current mtu is smaller than the uplink, which I think shall be kept or linked to a known issue until we fix it in 4.13.

Reply:
Yes, that is the place where we are talking about the bridges. Let's keep it for the moment and then we can figure out later how much bridge detail customers find helpful and take it from there.